src/main/java/wolox/training/models/Book.java

```diff
@@ -41,82 +44,38 @@
     @Column(nullable = false)
     private String isbn;
 
-    public long getId() {
-        return id;
-    }
-
-    public void setId(long id) {
-        this.id = id;
-    }
-
-    public String getGenre() {
-        return genre;
-    }
-
```

Comment:
Why are you deleting the getters but not the setters? I think the @Data annotation should provide both :)